src/topology/metric_space/basic.lean

```diff
@@ -1595,6 +1596,33 @@ lemma compact_iff_closed_bounded [t2_space α] [proper_space α] :
   exact compact_of_is_closed_subset (proper_space.compact_ball x r) hc hr
 end⟩
 
+lemma _root_.function.periodic.compact_of_continuous' {f : ℝ → α} {c : ℝ}
+  (hp : function.periodic f c) (hc : 0 < c) (hf : continuous f) :
+  is_compact (range f) :=
```

Comment:
What is `α` here?  I'm guessing, a metric space?  This lemma is true (same proof) for `α` a topological space.  Same for the next.